falcon/response.py

```diff
@@ -246,9 +246,21 @@ def set_cookie(self, name, value, expires=None, max_age=None,
 
     def unset_cookie(self, name):
         """Unset a cookie from the response
+        
+        Note:
+            This will instruct the browser to expire its copy of the cookie
+            if it has previously been set.
         """
-        if self._cookies is not None and name in self._cookies:
-            del self._cookies[name]
+        if self._cookies is None:
+            self._cookies = SimpleCookie()
+        if name not in self._cookies:
```

Comment:
Would it make sense to always set the value to the empty string, even if it was set elsewhere in the app to some value?